doc/api/tls.md

```diff
@@ -851,12 +851,12 @@ publicly trusted list of CAs as given in
   * `passphrase` {string} A string containing the passphrase for the private
     key or pfx.
   * `cert` {string|string[]|Buffer|Buffer[]} A string, `Buffer`, array of
-    strings, or array of `Buffer`s containing the certificate key of the server
-    in PEM format. (Required)
+    strings, or array of `Buffer`s containing the server's certificate chain
+    (server's certificate and intermediates) in PEM format. (Required)
```

Comment:
The intermediates should be optional and they are preferably placed in `ca`. IIRC, OpenSSL does accept both ways of adding them (concatenated vs. through `ca`).